app/bundles/CampaignBundle/Entity/EventRepository.php

```diff
@@ -244,41 +244,34 @@ public function getSearchCommands()
         return $this->getStandardSearchCommands();
     }
 
+    //Rewrite function
```

Comment:
It is not an optimization I just had to write the order of the query, because it was showing this error 

```
 ------ ----------------------------------------------------------------------- 
  Line   app/bundles/CampaignBundle/Entity/EventRepository.php                  
 ------ ----------------------------------------------------------------------- 
  282    QueryBuilder: [Syntax Error] line 0, col -1: Error: Expected Literal,  
         got end of string.                                                     
         DQL: SELECT e FROM MauticCampaignBundle:Event e INDEX BY e.id WHERE    
 ------ ----------------------------------------------------------------------- 
 
 ```

Reply:
Resolved in commit 1b8de9f7cd6486b520c8eb8b4e141e17b7498267